lab/torch/random.py

```diff
@@ -22,17 +22,25 @@ def global_random_state(_: TorchDType):
     else:
         parts = B.ActiveDevice.active_name.lower().split(":")
 
-        if len(parts) == 0 or parts[0] not in {"cuda", "gpu"}:
+        if len(parts) == 0 or parts[0] not in {"cuda", "gpu", "mps"}:
             raise RuntimeError(f'Unknown active device "{B.ActiveDevice.active_name}".')
 
-        # Ensure that the generators are available.
-        if len(torch.cuda.default_generators) == 0:
-            torch.cuda.init()
+        if parts[0] == "mps":
+            import torch.mps
```

Comment:
```suggestion
            import torch.mps as mps
```

I believe this breaks the global `import torch`. (See the failing test.) This should fix it.